packages/api/src/router/widgets/health-monitoring.ts

```diff
@@ -0,0 +1,52 @@
+import { observable } from "@trpc/server/observable";
+
+import type { HealthMonitoring } from "@homarr/integrations";
+import { createItemAndIntegrationChannel } from "@homarr/redis";
+
+import { createManyIntegrationMiddleware } from "../../middlewares/integration";
+import { createTRPCRouter, publicProcedure } from "../../trpc";
+
+export const healthMonitoringRouter = createTRPCRouter({
+  getHealthStatus: publicProcedure
+    .unstable_concat(createManyIntegrationMiddleware("query", "openmediavault"))
+    .query(async ({ ctx }) => {
+      return await Promise.all(
+        ctx.integrations.map(async (integration) => {
+          const channel = createItemAndIntegrationChannel<HealthMonitoring>("healthMonitoring", integration.id);
+          const data = await channel.getAsync();
+          const healthInfo = data?.data;
```

Comment:
Variable doesn't add any value. Remove it and use ``healthInfo: data?.data`` where you return the object